js/src/indexation.js

```diff
@@ -181,7 +181,9 @@ import ProgressBar from "./ui/progressBar";
 		if ( window.location.hash && window.location.hash.startsWith( "#start-indexation-" ) ) {
 			$( ".yoast-open-indexation" ).each( function() {
 				if ( window.location.hash.endsWith( $( this ).data( "settings" ) ) ) {
-					$( this ).click();
+					setTimeout( function() {
+						$( this ).click();
+					}.bind( this ), 500 );
```

Comment:
@andizer Does this mean that the empty modal will still be opened on slower sites that do not load within 500 ms?

Reply:
Hmm, I have no idea. Probably. But I am not sure. We can set this to 2000

Reply:
But then everyone with a fast website has to wait for 2 seconds for the modal to open 😓 
The time-out feels a bit hacky to me, but at the same time I have no better solution.

Reply:
Added a new fix 👍  Thanks!